diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter6/classes/SingleInitRule.kt

```diff
@@ -80,21 +95,21 @@ class SingleInitRule(configRules: List<RulesConfig>) : DiktatRule(
     }
 
     @Suppress("UnsafeCallOnNullableType", "TOO_LONG_FUNCTION")
-    private fun moveAssignmentsToProperties(properties: List<ASTNode>, initBlock: ASTNode) {
+    private fun moveAssignmentsToProperties(
+        properties: List<ASTNode>,
+        classParameters: List<String?>?,
```

Comment:
We shouldn't check for properties from the primary constructor, because they *can* be used to initialize properties:
```kotlin
class Foo(bar: Int) {
    val baz: Int = bar + 42
}
```